### ISSUE TITLE
Error bundling JS

### ISSUE THREAD

**Issue by imetallica** [galio-org/galio#140]
**Describe the bug**
I've added Galio to my project and when I try to run the app with Expo and simulator/Expo's mobile app, this is the error I'm getting when bundling:

```
SyntaxError: /Users/imetallica/Projects/Foco/mobile/node_modules/galio-framework/src/Text.js: Argument name clash (24:2)

  22 |   center,
  23 |   children,
> 24 |   style,
     |   ^
  25 |   styles,
  26 |   theme,
  27 |   ...rest
Failed building JavaScript bundle.
```

** Steps to reproduce **
1. Clone [this](https://github.com/creativetimofficial/argon-react-native) repository.
2. `npm install`
3. `npm run start`
4. Open the app in simulator or expo's mobile app

**Expected behavior**
Compile successfully

**Smartphone (please complete the following information):**
 - Device: iPhone Xr
 - Type of device: Simulator, Expo App on iOS
 - OS: iOS 13.1.2
 - React-Native version: 34
 - Expo version: 34.0.3

**Comment by ermalsh** [comment]
Due to PR #138 - 'style' is already declared at the top of the array object in the Text.js file.

**Comment by palingheorghe** [comment]
Solved in v0.6.3. [Release info](https://github.com/galio-org/galio/releases/tag/v0.6.3)
